src/test/java/io/debezium/operator/dependent/ServiceAccountDependentTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.operator.dependent;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.debezium.operator.DebeziumServer;
+import io.debezium.operator.DebeziumServerReconciler;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.fabric8.kubernetes.api.model.ServiceAccount;
+import io.javaoperatorsdk.operator.ReconcilerUtils;
+import io.javaoperatorsdk.operator.api.reconciler.Context;
+
+public class ServiceAccountDependentTest {
+
+    private final String managedSaName = "test-ds-sa";
+    private final String namespace = "debezium";
```

Comment:
Great. Just convert these two into constants (e.g. `private static final String MANAGED_SA_NAME`) and I will merge it.

Reply:
Done. 👍